### PR TITLE
Indent config improvements and color multiline bug fix

### PR DESCRIPTION
**Pull request by tonytangau**
Make indent configurable and also fix issue with color only works for the first line of multi-line logs.

### CODE REVIEW

File: src/BunyanDebugStream.coffee [hunk 1]
```diff
@@ -224,8 +223,8 @@ class BunyanDebugStream extends Writable
             #{date}#{processStr}#{levelPrefix}#{src}#{prefixes}#{applyColors message, colorsToApply}
         "
 
-        line += "\n#{INDENT}#{request}" if request?
-        line += "\n" + applyColors(values.join('\n'), colorsToApply) if values.length > 0
+        line += "\n#{@_indent}#{request}" if request?
+        line += "\n" + values.map((v) -> applyColors(v, colorsToApply)).join('\n') if values.length > 0
```
Comment (jwalton) [inline src/BunyanDebugStream.coffee]: Where are you running into trouble with colors?  The old version should work fine in a regular terminal.

**Comment by jwalton** [comment]
Published as v1.0.8.

File: src/BunyanDebugStream.coffee (hunk 1 above)
Comment (tonytangau) [inline src/BunyanDebugStream.coffee]: I'm using ConEmu, and it only display the first line of the parsed json object in color.
image

**Comment by jwalton** [comment]
Ah, I figured it had to be something slightly non-standard.  In a regular terminal, the colors will carry from one line to the next.  :)

**Pull request merged**